### PR TITLE
Only inject if content type is text/html

### LINKED ISSUE #90: Livereload inject causing buffering in start of stream
Hello! first big thanks for devd and modd!

I was working on a go web server that streams content back to the client when i noticed that browser got no data at all until a certain amount have been sent. First thought was to use `http.Flusher` but that did not works, then looked at TCP buffering and tcpdump but nothing. Then i tried without proxying thru devd and eveything worked.

After some digging in the devd source I figure out that it's caused most likely is the `CopyInject` code that buffers to be able to check for the marker.

Would it be possible to make the `CopyInject` smarter (only inject by content-type? be smarter when matching marker, send non-matching prefix etc?) or maybe make it possible enable/disable injection only for certain paths etc? something else?

### PR DESCRIPTION
**Pull request by wader**
Also to make streaming responses more useable enable flush timer and
implment Flush() for ResponseLogWriter to make flush actually work.

Fixes #90

### CODE REVIEW

File: route.go [hunk 1]
```diff
@@ -31,6 +32,7 @@ func (ep forwardEndpoint) Handler(prefix string, templates *template.Template, c
 	rp.Transport = &http.Transport{
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 	}
+	rp.FlushInterval = 200 * time.Millisecond
```
Comment (wader) [inline route.go]: To fast?

**Comment by wader** [comment]
This solves my issue, let me know what you think

**Comment by wader** [comment]
Hmm osx tests seem to fail for some reason, will have a look

**Comment by wader** [comment]
@cortesi hello! any opinions about this?

**Comment by cortesi** [comment]
@wader Hi there. Apologies for the slow response to this. I think the general approach here is good, and will knock out a number of corner cases where the injection misbehaves. I wonder if we're not going to end up extending this for more obscure html-ish content types (application/xhtml?).

File: route.go (hunk 1 above)
Comment (cortesi) [reply route.go]: That's a good question - I'd like to pull this out into a constant at the top of the file, so it's clear that we have a tunable parameter here that we need to be aware of.

Comment (wader) [reply route.go]: Yeap that sounds good

**Comment by wader** [comment]
@cortesi No problem and glad you liked it! Oh yeah good point about content type, so maybe `ContentType` should be a list or a regexp? Let me know if you want help looking at the corner cases.

**Pull request merged**